src/storage/index.js

```diff
@@ -1,12 +1,10 @@
-const { localStorage } = require('./local');
-
 const cache = {};
 
 class Storage {
   constructor(prefix, options = {}) {
     this._prefix = prefix;
 
-    const { customLocalStorage = localStorage } = options;
+    const { customLocalStorage } = options;
 
     this.localStorage = customLocalStorage;
```

Comment:
why do we need `this.localStorage` if you removed it being used in `set` and `get`?

Reply:
we don't, removed